okcurl/src/test/kotlin/okhttp3/curl/OkcurlTest.kt

```diff
@@ -16,9 +16,18 @@
 package okhttp3.curl
 
 import com.github.ajalt.clikt.core.main
+import kotlin.test.BeforeTest
 import kotlin.test.Test
+import okhttp3.TestUtil.assumeNotWindows
 
 class OkcurlTest {
```

Comment:
This whole test seems redundant with `MainTest` which already parses, binds, and runs the `Main` command many times.

Reply:
https://github.com/square/okhttp/pull/8948